### ISSUE TITLE
github repo authentication

### ISSUE THREAD

**Issue by CyberKavaju** [microsoft/vscode#137229]
Issue Type: <b>Bug</b>

I am getting following error while pulling or pushing in vscode integrated terminal.

This error is showing since las update.

I am using vscode in ubuntu 20.04 LTS
ERROR------------------------------------
remote: Repository not found.
fatal: Authentication failed for 'https://github.com/<username>/<repo>.git/'
----------------------------------------------
I am able to pull or push from normal terminal but I have to use username and token everytime I do anything.

What should I do to get rid of this?

VS Code version: Code 1.62.2 (3a6960b964327f0e3882ce18fcebd07ed191b316, 2021-11-11T20:51:30.084Z)
OS version: Linux x64 5.11.0-40-generic
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz (4 x 799)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|7, 6, 6|
|Memory (System)|7.64GB (0.29GB free)|
|Process Argv|--unity-launch --crash-reporter-id 1fc05c4a-1509-4ae5-b3b9-b2abbb3b9ebb|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (25)</summary>

Extension|Author (truncated)|Version
---|---|---
better-comments|aar|2.1.0
aw-watcher-vscode|act|0.5.0
bootstrap5-vscode|Anb|0.2.0
vscode-database|baj|2.2.3
vscode-intelephense-client|bme|1.7.1
vscode-mysql-client2|cwe|4.2.0
vscode-eslint|dba|2.2.2
gitlens|eam|11.6.1
php-debug|fel|1.22.0
php-intellisense|fel|2.3.14
php-pack|fel|1.0.2
auto-close-tag|for|0.5.13
auto-rename-tag|for|0.1.9
copilot|Git|1.7.3822
html-tag-wrapper|hwe|0.2.3
restructuredtext|lex|167.0.0
ftp-sync|luk|0.3.9
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.3
sublime-keybindings|ms-|4.0.10
php-docblocker|nei|2.6.1
LiveServer|rit|5.6.1
vscode-fontawesome-gallery|tom|0.0.5
simple-rst|tro|1.5.0
JavaScriptSnippets|xab|1.8.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627cf:30244335
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30384386
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis200:30386379
vsaa593:30376534
vssld246:30386377

```

</details>

**Comment by gjsjohnmurray** [comment]
Probably a dup. See https://github.com/microsoft/vscode/issues/137161#issuecomment-968726316 for pointer on how to downgrade to 1.62.0.

**Comment by gjsjohnmurray** [comment]
/duplicate of #137183 which is scheduled to be fixed in upcoming recovery release (1.62.3)
